ci/deps/azure-macos-36.yaml

```diff
@@ -26,6 +26,7 @@ dependencies:
   - pytables
   - python-dateutil==2.6.1
   - pytz
+  - scipy>=1.1
```

Comment:
hmm, we *should* be skipping tests if the deps are not installed, which test is the issue?

Reply:
> hmm, we _should_ be skipping tests if the deps are not installed, which test is the issue?

```
pandas.tests.frame.methods.test_cov_corr.TestDataFrameCorrWith
test_moments_rolling.TestRollingMomentsConsistency
pandas.tests.frame.methods.test_cov_corr.TestDataFrameCorrWith
pandas.tests.test_nanops.TestnanopsDataFrame
pandas.tests.test_nanops.TestnanopsDataFrame
pandas.tests.frame.methods.test_cov_corr.TestDataFrameCorrWith
pandas.tests.series.test_timeseries.TestTimeSeries
```

For the macOS build, at least.

Reply:
then these should have a skip in place